### ISSUE TITLE
Running pretrained bert model causes error

### ISSUE THREAD

**Issue by zzj0402** [chengchingwen/Transformers.jl#11]
```
bert_model, wordpiece, tokenizer = pretrain"bert-uncased_L-12_H-768_A-12"
ERROR: TypeError: in keyword argument cookiejar, expected Dict{String,Set{HTTP.Cookies.Cookie}}, got Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}
Stacktrace:
 [1] (::getfield(HTTP, Symbol("#kw##request")))(::NamedTuple{(:reached_redirect_limit, :cookies, :cookiejar),Tuple{Bool,Bool,Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}}}, ::typeof(HTTP.request), ::Type{HTTP.CookieRequest.CookieLayer{HTTP.MessageRequest.MessageLayer{HTTP.RetryRequest.RetryLayer{HTTP.ExceptionRequest.ExceptionLayer{HTTP.ConnectionRequest.ConnectionPoolLayer{HTTP.StreamRequest.StreamLayer{Union{}}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}) at ./none:0
 [2] #request#1(::Base.Iterators.Pairs{Symbol,Any,Tuple{Symbol,Symbol,Symbol},NamedTuple{(:reached_redirect_limit, :cookies, :cookiejar),Tuple{Bool,Bool,Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}}}}, ::Function, ::Type{HTTP.BasicAuthRequest.BasicAuthLayer{HTTP.CookieRequest.CookieLayer{HTTP.MessageRequest.MessageLayer{HTTP.RetryRequest.RetryLayer{HTTP.ExceptionRequest.ExceptionLayer{HTTP.ConnectionRequest.ConnectionPoolLayer{HTTP.StreamRequest.StreamLayer{Union{}}}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}) at /home/zzj0402/.julia/packages/HTTP/ZggHU/src/BasicAuthRequest.jl:28
 [3] (::getfield(HTTP, Symbol("#kw##request")))(::NamedTuple{(:reached_redirect_limit, :cookies, :cookiejar),Tuple{Bool,Bool,Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}}}, ::typeof(HTTP.request), ::Type{HTTP.BasicAuthRequest.BasicAuthLayer{HTTP.CookieRequest.CookieLayer{HTTP.MessageRequest.MessageLayer{HTTP.RetryRequest.RetryLayer{HTTP.ExceptionRequest.ExceptionLayer{HTTP.ConnectionRequest.ConnectionPoolLayer{HTTP.StreamRequest.StreamLayer{Union{}}}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}) at ./none:0
 [4] #request#1(::Int64, ::Bool, ::Base.Iterators.Pairs{Symbol,Any,Tuple{Symbol,Symbol},NamedTuple{(:cookies, :cookiejar),Tuple{Bool,Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}}}}, ::Function, ::Type{HTTP.RedirectRequest.RedirectLayer{HTTP.BasicAuthRequest.BasicAuthLayer{HTTP.CookieRequest.CookieLayer{HTTP.MessageRequest.MessageLayer{HTTP.RetryRequest.RetryLayer{HTTP.ExceptionRequest.ExceptionLayer{HTTP.ConnectionRequest.ConnectionPoolLayer{HTTP.StreamRequest.StreamLayer{Union{}}}}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}) at /home/zzj0402/.julia/packages/HTTP/ZggHU/src/RedirectRequest.jl:24
 [5] (::getfield(HTTP, Symbol("#kw##request")))(::NamedTuple{(:cookies, :cookiejar),Tuple{Bool,Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}}}, ::typeof(HTTP.request), ::Type{HTTP.RedirectRequest.RedirectLayer{HTTP.BasicAuthRequest.BasicAuthLayer{HTTP.CookieRequest.CookieLayer{HTTP.MessageRequest.MessageLayer{HTTP.RetryRequest.RetryLayer{HTTP.ExceptionRequest.ExceptionLayer{HTTP.ConnectionRequest.ConnectionPoolLayer{HTTP.StreamRequest.StreamLayer{Union{}}}}}}}}}}, ::String, ::HTTP.URIs.URI, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}) at ./none:0
 [6] #request#4(::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}, ::Nothing, ::Base.Iterators.Pairs{Symbol,Any,Tuple{Symbol,Symbol},NamedTuple{(:cookies, :cookiejar),Tuple{Bool,Array{Dict{String,Set{HTTP.Cookies.Cookie}},1}}}}, ::Function, ::String, ::String, ::Array{Pair{SubString{String},SubString{String}},1}, ::Array{UInt8,1}) at /home/zzj0402/.julia/packages/HTTP/ZggHU/src/HTTP.jl:314
 [7] #request at ./none:0 [inlined] (repeats 2 times)
 [8] download_gdrive(::String, ::String) at /home/zzj0402/.julia/packages/Transformers/2BZJs/src/datasets/download_utils.jl:56
 [9] run_fetch(::typeof(Transformers.Datasets.download_gdrive), ::String, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution_automatic.jl:99
 [10] #download#13(::String, ::Nothing, ::Bool, ::Function, ::DataDeps.DataDep{String,String,typeof(Transformers.Datasets.download_gdrive),typeof(identity)}, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution_automatic.jl:78
 [11] download(::DataDeps.DataDep{String,String,typeof(Transformers.Datasets.download_gdrive),typeof(identity)}, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution_automatic.jl:70
 [12] handle_missing(::DataDeps.DataDep{String,String,typeof(Transformers.Datasets.download_gdrive),typeof(identity)}, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution_automatic.jl:10
 [13] _resolve(::DataDeps.DataDep{String,String,typeof(Transformers.Datasets.download_gdrive),typeof(identity)}, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution.jl:83
 [14] resolve(::DataDeps.DataDep{String,String,typeof(Transformers.Datasets.download_gdrive),typeof(identity)}, ::String, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution.jl:29
 [15] resolve(::String, ::String, ::String) at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution.jl:54
 [16] resolve at /home/zzj0402/.julia/packages/DataDeps/gbyoa/src/resolution.jl:73 [inlined]
 [17] #load_pretrain#3(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::String) at /home/zzj0402/.julia/packages/Transformers/2BZJs/src/pretrain/Pretrain.jl:64
 [18] load_pretrain(::String) at /home/zzj0402/.julia/packages/Transformers/2BZJs/src/pretrain/Pretrain.jl:53
 [19] top-level scope at none:0
```

**Comment by chengchingwen** [comment]
Hi @zzj0402,

Looks like there are some error when downloading the model. 
What version of HTTP and Transformers are you using?

**Comment by zzj0402** [comment]
> Hi @zzj0402,
> 
> Looks like there are some error when downloading the model.
> What version of HTTP and Transformers are you using?

Not sure, just followed tutorial with a fresh install of Julia 1.4

**Comment by freddycct** [comment]
I had this problem with v0.11, you have to use the latest v0.13

**Comment by chengchingwen** [comment]
Ah, right. The downloading code in v0.1.1 is outdated due to some changes in HTTP.jl and google drive downloading behavior. v0.1.3 should be fine

**Comment by chengchingwen** [comment]
This should be fix with version higher then v0.1.3
